doc/source/ScriptingAPI/SpecialVariables/index.md

```diff
@@ -0,0 +1,32 @@
+
+# Special Variables #
+
+The Gaffer API has some special reserved utility variables for use in expressions and code input fields.
+
+
+## The “script” Variable ##
+
+In Gaffer scripts, the variable used to invoke the root of the script is the `script` variable. When building the node graph of your script, nodes are added using this variable.
+
+
+## The “parent” Variable ##
+
+The `parent` variable is used to invoke the containing node or the root of the script, depending on the context. This variable is primarily used in serialized code, to avoid scope issues when transferring code between [script files](../ScriptFiles/index.md), Reference and Box nodes <!-- TODO: box and reference article links-->, and the [application's code input fields](../../NodeGraphScripting/ScriptEditor/index.md).
```

Comment:
When copy-pasting serialized graphs into the _Script Editor_, `parent` will be the default variable, which was what I was trying to convey here. If not for that, the distinction between `parent` and `script` would be arbitrary.

Reply:
Ah, good point about running serialized nodes from the script editor - I couldn't remember why that parent variable even existed in the script editor.  So "application's code input fields" here just means "Script Editor"?

Expressions also have a parent variable, which means something similar, but not really identical.  I'm only really thinking about the expression case, because you mention context below, which is an expression special variable, and not an API special variable.